<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="messageLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>391</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>391</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="listenButton">
       <property name="text">
        <string>Listen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="connectButton">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sendFileButton">
       <property name="text">
        <string>Send File</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>436</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>listenButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onListen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>11</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>connectButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onConnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sendFileButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSendFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onListen()</slot>
  <slot>onConnect()</slot>
  <slot>onSendFile()</slot>
 </slots>
</ui>
